lottie/src/main/java/com/airbnb/lottie/model/KeyPath.java

```diff
@@ -35,12 +36,17 @@
  *        new KeyPath("*", "Body", Left Hand", "Fill");
  *     Match anything with the name Fill:
  *        new KeyPath("**", "Fill");
+ *     Target the the root composition layer:
+ *        KeyPath.COMPOSITION
  *
  *
  * NOTE: Content that are part of merge paths or repeaters cannot currently be resolved with
  * a {@link KeyPath}. This may be fixed in the future.
  */
 public class KeyPath {
+  // A singleton KeyPath that targets on the root composition layer.
+  // This is useful if you want to apply transformer to the animation as a whole.
```

Comment:
nit: use the class javadoc style so you get the right documentation popup in Android Studio

Reply:
Good catch. I thought they were same in term of inline comments. Just realized the difference.